scanpy/external/pp/_magic.py

```diff
@@ -124,9 +135,12 @@ def magic(
             'git+git://github.com/KrishnaswamyLab/MAGIC.git#subdirectory=python`'
         )
     else:
-        __version__ = tuple([int(v) for v in __version__.split(".")[:2]])
-        if not __version__ >= (1, 5):
+        version = tuple([int(v) for v in __version__.split(".")[:2]])
+        min_version = (2, 0)
+        if not version >= min_version:
             raise ImportError(
+                'scanpy requires magic-impute >= '
+                f'v{min_version[0]}.{min_version[1]} (detected: v{__version__}). '
```

Comment:
```suggestion
                f'v{min_version[0]}.{min_version[1]} (detected: v{version}). '
```

Reply:
`__version__` is the string version, `version` is the parsed tuple. I could do `v{version[0]}.{version[1]}` if you prefer.

Reply:
`__version__` doesn’t exist, you changed it.

Reply:
Ah, whoops, I just saw you renaming it, not that `__version__` still exists. Apologies